<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestAffecterBI</class>
 <widget class="QMainWindow" name="GestAffecterBI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>571</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Information Personel</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>491</width>
       <height>364</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNom">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Prenom: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPrenom">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Numero de Pièce :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditNumPiece">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Contact: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditContact">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Email: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditEMail">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>Sexe :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxSexe">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Homme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Femme</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date de Naissance:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDateEdit" name="dateEditDateDeNaissance">
        <property name="font">
         <font>
          <family>Vani</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>Id Bien Immobilier: </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditIdBien">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>Type de Location :</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboBoxTypeLocation">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Long Terme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Journalière</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonAnnuler">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonAffecter">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Affecter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
